ci-operator/config/openshift/kedacore-keda/openshift-kedacore-keda-main.yaml

```diff
@@ -6,8 +6,34 @@ base_images:
 build_root:
   from_repository: true
 images:
-- from: base
+- build_args:
+  - name: GIT_COMMIT
```

Comment:
I really wish there was a magic variable to inject the git commit here. I know CI knows it, I just don't know how to connect the tubes. 

Why this is necessary: 

- The prometheus test also [compares its git HEAD to the metrics adapter pod](https://github.com/openshift/kedacore-keda/blob/acae035f753afece47747af5d49e40cddf4e114f/tests/sequential/prometheus_metrics/prometheus_metrics_test.go#L635) to make sure the metrics adapter matches the same build as our tests 
- The test pod knows its git commit because it has the source that we copied into it 
- The metrics adapter pod gets its git commit version from the arg when it is built 
  - The `GIT_COMMIT` arg does get supplied during the build when we build outside of CI [using the Makefile](https://github.com/openshift/kedacore-keda/blob/acae035f753afece47747af5d49e40cddf4e114f/Makefile#L218)
  - But the  `GIT_COMMIT` arg **does not** get supplied when we build in CI -- it just uses the default [in the Dockerfile](https://github.com/openshift/kedacore-keda/blob/acae035f753afece47747af5d49e40cddf4e114f/Dockerfile.webhooks#L5) which is "HEAD", apparently. 

 So I'm injecting a dummy string "dummy-ci-commit-value" for now so the test can at least know that it's talking to something else that's built in CI and we're not pulling some upstream image or something.